### DISCUSSION TITLE
Plugin directories disappeared from master

### DISCUSSION THREAD

**Discussion by EandrewJones** [dotansimha/graphql-code-generator#8655] [Q&A] [answered]
I was previously working on a fix for an issue in the `typescript-react-query` plugin. The other day I went to continue that work and pull updates from master but noticed a bunch of the plugin src code directories are missing from master now, including the typescript-react-query plugin.

I've scanned through announcements, recent releases, etc. to find an explanation but feel like I'm missing something that I shouldn't be. 

Thanks for update.

**Comment by charlypoly** [comment, marked as answer]
Hi @EandrewJones,

As announced in the [roadmap](https://github.com/dotansimha/graphql-code-generator/issues/8296#:~:text=Introduction%20of%20the%20%E2%80%9Ccommunity%20plugins%E2%80%9D), most of existing plugins have been moved to a new repository: [dotansimha/graphql-code-generator-community](https://github.com/dotansimha/graphql-code-generator-community).

All the pending PRs have been pinged accordingly, for example, https://github.com/dotansimha/graphql-code-generator/pull/8408#issuecomment-1305392798

To migrate your local work/branch to the new repo, please follow these instructions:


From your current PR branch:
- Fork the https://github.com/dotansimha/graphql-code-generator-community repository
- git remote add codegen-community <GIT_URL_OF_YOUR_COMMUNITY_REPO_FORK>
- git fetch codegen-community
- git push codegen-community

Then, open the PR on [`dotansimha/graphql-code-generator-community`](https://github.com/dotansimha/graphql-code-generator-community).

Thank you!

**Comment by EandrewJones** [comment]
Many thanks. I guess I read everything except the roadmap.
